cobra.go

```diff
@@ -40,14 +40,6 @@ var initializers []func()
 // Set this to true to enable it
 var EnablePrefixMatching bool = false
 
-// enables an information splash screen on Windows if the CLI is started from explorer.exe.
-var EnableWindowsMouseTrap bool = true
```

Comment:
You can replace the code with an empty function. Now that I think about it it should be more like this (sans better name. Don't like it either):

```
var preExecHook = preExecHookWin

func preExecHookWin() {...}
```

This way a user can override the function with a custom one - or replace it with nil to disable it.